spec/load_spec.rb

```diff
@@ -7,17 +7,19 @@
 
   it "provides require_all functionality but using 'load' instead of 'require'" do
     require_all File.dirname(__FILE__) + '/fixtures/resolvable'
-    C.new.should be_cool
+    expect(C.new).to be_cool
 
     class C
+      remove_method :cool?
```

Comment:
`require_all` on line 9 defines `C#cool?`. To allow redefinition without warnings on line 14, `#cool?` needs to be removed first. Then, on line 19, it must be removed again to allow `load_all` on line 21 to define it again. In summary, `#cool?` is defined three times, so has to be removed two times to avoid warnings that the method is being redefined.

This has the same semantics as the original spec because `require` only checks whether the file has been loaded, not whether its definitions are still present.